src/common-components/EnterpriseSSO.jsx

```diff
@@ -32,21 +32,20 @@ const EnterpriseSSO = (props) => {
       <div className="d-flex justify-content-center m-4">
         <div className="d-flex flex-column">
           <div className="mw-450">
-            <h3>Sign in</h3>
             <Form className="m-0">
               <p>{intl.formatMessage(messages['enterprisetpa.title.heading'], { providerName: tpaProvider.name })}</p>
               <Button
                 id={tpaProvider.id}
                 key={tpaProvider.id}
                 type="submit"
-                variant="primary"
-                className="btn-tpa"
+                link="primary"
```

Comment:
This is not what I meant, you need to change the value of variant to link, `variant="link"`.